### PR TITLE
move most of install html into twig templates and apply bootstrap styles

### PR DESCRIPTION
**Pull request by orthagh**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

Initialy i started (it slipped a little :p) the work by wanted to fix the language selector (a select2 one) present at first page of install/update process and i found css styles was not loaded.

I added a var `$skip_db_check` in front/css.php to have scss compile feature available for installation (This is most important change), `includes.php` check this var before checking existence of DB.

And i moved most html parts of installation into twig templates.
I didn't touch a lot of update process (just the final page and buttton styles)

I found the following errors i didn't understant (yet) :
```
[2021-09-09 11:49:09] glpiphplog.DEBUG: Html::compileScss() in /var/www/html/glpi/10.0.git/inc/html.class.php line 6792
Compile css/style_install.scss  {"user":"@DESKTOP-CUCL3BV"} 
[2021-09-09 11:49:28] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpidate_format" in /var/www/html/glpi/10.0.git/inc/toolbox.class.php at line 2910
  Backtrace :
  inc/html.class.php:6449                            Toolbox::getDateFormat()
  install/install.php:61                             Html::getCoreVariablesForJavascript()
  install/install.php:547                            header_html()
  
[2021-09-09 11:49:28] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "" in /var/www/html/glpi/10.0.git/inc/toolbox.class.php at line 2910
  Backtrace :
  inc/html.class.php:6449                            Toolbox::getDateFormat()
  install/install.php:61                             Html::getCoreVariablesForJavascript()
  install/install.php:547                            header_html()
```

### CODE REVIEW

**Comment by orthagh** [comment]
To note, the above log entries are already presents in `feat/modern-ui` branch

File: inc/telemetry.class.php [hunk 1]
```diff
@@ -343,7 +343,9 @@ public static final function generateRegistrationUuid() {
    public static function getViewLink() {
       global $CFG_GLPI;
 
-      $out = "<a id='view_telemetry' href='{$CFG_GLPI['root_doc']}/ajax/telemetry.php'>" . __('See what would be sent...') . "</a>";
+      $out = "<a id='view_telemetry' href='{$CFG_GLPI['root_doc']}/ajax/telemetry.php' class='btn btn-sm btn-info mt-2'>
```
Comment (cedric-anne) [inline inc/telemetry.class.php:346]: Text in Telemetry dialog is not readable.

image

Comment (cedric-anne) [reply inc/telemetry.class.php:346]: Fixed.

**Comment by cedric-anne** [comment]
> I found the following errors i didn't understant (yet) :
> 
> ```
> [2021-09-09 11:49:09] glpiphplog.DEBUG: Html::compileScss() in /var/www/html/glpi/10.0.git/inc/html.class.php line 6792
> Compile css/style_install.scss  {"user":"@DESKTOP-CUCL3BV"} 
> [2021-09-09 11:49:28] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "glpidate_format" in /var/www/html/glpi/10.0.git/inc/toolbox.class.php at line 2910
>   Backtrace :
>   inc/html.class.php:6449                            Toolbox::getDateFormat()
>   install/install.php:61                             Html::getCoreVariablesForJavascript()
>   install/install.php:547                            header_html()
>   
> [2021-09-09 11:49:28] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "" in /var/www/html/glpi/10.0.git/inc/toolbox.class.php at line 2910
>   Backtrace :
>   inc/html.class.php:6449                            Toolbox::getDateFormat()
>   install/install.php:61                             Html::getCoreVariablesForJavascript()
>   install/install.php:547                            header_html()
> ```

I have fix them in 3a402b2e583ec0ba9fb92088b72b56f7832b1576

File: inc/config.php [hunk 2]
```diff
@@ -60,7 +60,7 @@
 
 Config::detectRootDoc();
```
[the remark this answers is not in the document]
Comment (cedric-anne) [reply inc/config.php:62]: In fact, as there is already variabkles for other checks, declared in front/css.php (see following snippet), proposed solution is OK.
```php
$_GET["donotcheckversion"]   = true;
$dont_check_maintenance_mode = true;
```

**Review by trasher — APPROVED** [review]

**Pull request merged**